basic.go

```diff
@@ -26,13 +28,24 @@ type emitter struct {
 	typ     reflect.Type
 	closed  int32
 	dropper func(reflect.Type)
+	b       *basicBus
 }
 
 func (e *emitter) Emit(evt interface{}) error {
 	if atomic.LoadInt32(&e.closed) != 0 {
 		return fmt.Errorf("emitter is closed")
 	}
 	e.n.emit(evt)
+
+	// emit on the wildcard subscription
+	e.b.lk.RLock()
+	n, ok := e.b.nodes[wildCardTypeElem]
+	e.b.lk.RUnlock()
```

Comment:
I'd prefer to avoid the global read lock and the map access, if at all possible.

Any reason not to stash the wildcard emitter's node in an `atomic.Value` inside the bus's struct? We'd need to do a little more special casing but this code would become:

```go
n, _ := e.b.wildcard.Load().(*node)
if !ok {
  return nil
}
```

Reply:
Tackled differently in #40.